tests/cli/test_main_clean.py

```diff
@@ -69,119 +68,124 @@ def has_pkg(name: str, contents: Iterable[str | Path]) -> bool:
 # conda clean --force-pkgs-dirs
 def test_clean_force_pkgs_dirs(
     clear_cache,
+    test_recipes_channel: Path,
     conda_cli: CondaCLIFixture,
     tmp_env: TmpEnvFixture,
+    tmp_pkgs_dir: Path,
 ):
-    pkg = "zlib"
+    pkg = "small-executable"
```

Comment:
This is served by `test_recipes_channel`, right?

Reply:
Yup